setup.py

```diff
@@ -1,25 +1,33 @@
+#!/usr/bin/env python
+
 import setuptools
+from frost import SOURCE_URL, VERSION
 
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+install_requires = [
+    line.split("==")[0] for line in open("requirements.txt", "r") if ".git" not in line
+]
+
 setuptools.setup(
     name="frost",
-    version="0.3.1",
+    version=VERSION,
     author="Firefox Operations Security Team (foxsec)",
     author_email="foxsec+frost@mozilla.com",
     description="tests for checking that third party services the Firefox Operations Security or foxsec team uses are configured correctly",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/mozilla/frost",
+    url=SOURCE_URL,
     license="MPL2",
     packages=setuptools.find_packages(),
+    install_requires=install_requires,
     classifiers=[
         "Natural Language :: English",
         "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
     ],
-    python_requires=">=3.6",
+    python_requires=">=3.8",
```

Comment:
This will make https://github.com/mozilla/frost/blob/master/Makefile#L10-L14 kind of strange. Can we remove the min python version checking in Makefile now that this is here?

Reply:
I think so, do we have anything using Python 3.6 still?

Reply:
Nope, Jenkins runs are all on 3.8 now.

Reply:
updated